src/hooks/getGlobalStats.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react'
+
+const GetGlobalStats = () => {
+  const [circulatingSupply, setcirculatingSupply] = useState('')
+  const [totalSupply, setTotalSupply] = useState('')
+  const [totalPaidToHolders, setTotalPaidToHolders] = useState('')
+  const [burntSupply, setBurntSupply] = useState('')
+  const [marketCap, setMarketCap] = useState('')
+
+  // Function for inserting commas in the number
+  function numberWithCommas(x) {
+    return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')
+  }
+
+  // fetch CirculatingSupply from blockchain
+  const fetchCirculatingSupply = async () => {
+    const response = await '2120000000'
+    setcirculatingSupply(await numberWithCommas(response))
```

Comment:
we're probably gonna want these to return a big number

and we'll probably need a formatting utility function for formatting